aria/modeling/service_template.py

```diff
@@ -1544,11 +1547,24 @@ def as_raw(self):
 
     def instantiate(self, container):
         from . import models
+        from ..orchestrator import context
+        plugin = None
+        if self.plugin_specification is not None:
+            try:
+                workflow_context = context.workflow.current.get()
+                plugin = self.plugin_specification.find_plugin(workflow_context.model.plugin.list())
+            except context.exceptions.ContextException:
+                pass
+                # TODO
```

Comment:
Yes, we need to talk about this. :) I don't know how else to find other models. Is there a current context for SQLAlchemy?